src/client/scm_estimator/README.md

```diff
@@ -0,0 +1,199 @@
+
+# SCM/NVMe Estimation Tool
+
+The vos_scm_estimator.py tool estimates the utilization of the Storage Class Memory (SCM) required for DAOS deployments. DAOS uses the <a href="https://github.com/daos-stack/daos/blob/master/src/vos/README.md">Versioning Object Store (VOS)</a> to keep track of the DAOS objects metadata.
+There are three options to feed the tool with the description of the items that will be stored in the <a href="https://github.com/daos-stack/daos/blob/master/src/client/dfs/README.md">DAOS File system</a>.
+
+To run the tool, first setup the paths
+
+```
+$ source ./utils/sl/setup_local.sh
```

Comment:
This only works in a development environment though.

How to these instructions translate for somebody who installs DAOS from RPMs and where everything lives in the system paths (i.e. `/usr/*`)?

Reply:
I just installed the DAOS rpms and the tool is available on the PATH. There is no need to run the source command. However, there is an issue with one of the shared libraries. It seems to be on a different directory than development environment. I am working on a fix